docs/content/usage/plugins/_index.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Plugins"
+date = 2023-08-31T14:39:18+00:00
+weight = 10
+chapter = false
+pre = '<i class="fas fa-puzzle-piece"></i> '
++++
+
+## Overview
+
+The Infection Monkey is extensible! You can install plugins to enhance the functionality of the Infection Monkey.
+
+Plugins page consists of three tabs:
+
+- **Available Plugins** - A table of plugins that are available for download and installation.
+- **Installed Plugins** - A table of plugins that are currently installed on the Monkey Island.
+- **Upload New Plugin** - A form to upload a new plugin to the Monkey Island. This is useful if you
```

Comment:
How do I upgrade a plugin to a newer version?

Reply:
If you don't know this, then we need to fix UX, not docs